usage-demo/README.md

```diff
@@ -0,0 +1,17 @@
+# JSpecify Reference Checker Usage Demo
+
+This is a simple demonstration for how a gradle project can use the JSpecify Reference Checker.
+
+Until the JSpecify Reference Checker is released to Maven Central, on must first run:
+
+````
+./gradlew PublishToMavenLocal
```

Comment:
I would have guessed that _this_ command would need to use `..` and the one on line 14 would need to use `.`. I haven't tested that theory, but does that sound right?

Reply:
I think the number of dots is correct. I added more explanations to hopefully make it clearer.

Reply:
Yes, thank you, that completely clarified things for me.